### PR TITLE
DEX-184 Rebrand stakepool repository

### PR DESCRIPTION
**Pull request by harzo**

### CODE REVIEW

**Comment by jksf** [comment]
I think it should be named `EXCS`

```
log.go
51:     log                 = backendLog.Logger("DCRS")
67:     "DCRS": log,
```

File: views/settings.html [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
       <p><strong>To start staking, you need to submit a public key address.  This can be
-      done automatically by the GUI wallet Decrediton (see below).
+      done automatically by the GUI wallet Exilibrium (see below).
       You may also proceed by following the command-line instructions on the
       <a href="/address">Address Page</a>.</strong></p><br>
       {{end}}
-        <p><strong>The official Decred GUI wallet
-        <a href="https://decred.org/downloads/#decrediton">Decrediton</a>
+        <p><strong>The official EXCCoin GUI wallet
+        <a href="https://excc.org/downloads/#exilibrium">Exilibrium</a>
```
Comment (jwjanus) [inline views/settings.html]: probably it should be excc.co

File: views/settings.html [hunk 2]
```diff
@@ -10,18 +10,18 @@
...
         is the recommended way to use stake pools.  All you need to get started
         is to copy and paste the API Token below when prompted.  A more
         comprehensive guide is available for
-        <a href="https://docs.decred.org/getting-started/user-guides/decrediton-setup/">Decrediton</a>.
+        <a href="https://docs.excc.org/getting-started/user-guides/exilibrium-setup/">Exilibrium</a>.
         There is also a more general overview of the staking process available
-				<a href="https://docs.decred.org/mining/proof-of-stake/#automatic-purchasing-of-tickets" target="_blank">here</a>.</strong></p>
+				<a href="https://docs.excc.org/mining/proof-of-stake/#automatic-purchasing-of-tickets" target="_blank">here</a>.</strong></p>
```
Comment (jwjanus) [inline views/settings.html]: excc.co

File: views/settings.html [hunk 3]
```diff
@@ -10,18 +10,18 @@
...
       {{end}}
-        <p><strong>The official Decred GUI wallet
-        <a href="https://decred.org/downloads/#decrediton">Decrediton</a>
+        <p><strong>The official EXCCoin GUI wallet
+        <a href="https://excc.org/downloads/#exilibrium">Exilibrium</a>
         is the recommended way to use stake pools.  All you need to get started
         is to copy and paste the API Token below when prompted.  A more
         comprehensive guide is available for
-        <a href="https://docs.decred.org/getting-started/user-guides/decrediton-setup/">Decrediton</a>.
+        <a href="https://docs.excc.org/getting-started/user-guides/exilibrium-setup/">Exilibrium</a>.
```
Comment (jwjanus) [inline views/settings.html]: excc.co

Comment (jksf) [reply views/settings.html]: Yes, we don't have excc.org domain.

**Pull request merged**